dl/src/main/scala/com/intel/analytics/bigdl/utils/Engine.scala

```diff
@@ -322,14 +349,93 @@ object Engine {
       }
       Some(sc)
     } else {
-      physicalCoreNumber = cores
+      _onSpark = false
```

Comment:
Yes, we should add an option to bigdl.sh; shall we also check if bigdl.sh is used by the user?